src/Kunstmaan/GeneratorBundle/Resources/SensioGeneratorBundle/skeleton/defaultsite/Resources/views/Pages/HomePage/view.html.twig

```diff
@@ -11,3 +11,18 @@
     {{ render_pagetemplate(page) }}
 {% endblock %}
 
+{% block footerscripts %}
+    <!-- Google search integration -->
+    <script type="application/ld+json">
+        {
+          "@context": "http://schema.org",
+          "@type": "WebSite",
+          "url": "{{ url("_slug", {url: '/'}) }}",
+          "potentialAction": {
+            "@type": "SearchAction",
+            "target": "{{ get_url_by_internal_name('search', app.request.locale) }}?query={search_term_string}",
```

Comment:
Not every site will have an active search page. You should only add this when a search page is actually available.